### PR TITLE
Jazzmin admin theme

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
*trumpets*

### CODE REVIEW

File: docker-app/qfieldcloud/settings.py [hunk 1]
```diff
@@ -575,3 +576,24 @@ def before_send(event, hint):
...
+    # Logo to use for your site, must be present in static files, used for brand on top left
+    "site_logo": "img/opengis_powering_qfc.png",
+    # Logo to use for your site, must be present in static files, used for login form logo (defaults to site_logo)
+    "login_logo": "img/opengis_powering_qfc.png",
+    # Welcome text on the login screen
+    "welcome_sign": "Welcome to QFieldCloud",
+    # Copyright on the footer
+    "copyright": "Opengis.ch",
+    # Custom css
+    "custom_css": "css/login.css",
```
Comment (suricactus) [inline docker-app/qfieldcloud/settings.py:598]:
```suggestion
    # Page title. Defaults to `current_admin_site.site_title` if empty.
    "site_title": "QFieldCloud: Admin",
    # Site header title (max 19 chars). Defaults to  `current_admin_site.site_header` if empty.
    "site_header": "QFieldCloud",
    # Brand name (max 19 chars). Defaults to  `current_admin_site.site_header` if empty.
    "site_brand": "QFieldCloud",
    # Favicon filename. Defaults to `site_logo` if empty.
    "site_icon": "favicon.ico",
    # Logo filename.
    "site_logo": "img/opengis_powering_qfc.png",
    # Sign-in form logo filename. Defaults to `site_logo` if empty.
    "login_logo": "img/opengis_powering_qfc.png",
    # Sign-in form welcome text.
    "welcome_sign": "Welcome to QFieldCloud",
    # Copyright on the footer.
    "copyright": "OPENGIS.ch",
    # Additional custom CSS file for the Django Admin pages.
    "custom_css": "css/login.css",
```

File: docker-app/qfieldcloud/settings.py [hunk 2]
```diff
@@ -67,6 +67,7 @@
 # Application definition
 INSTALLED_APPS = [
     # django contrib
+    "jazzmin",  # admin theme
```
Comment (suricactus) [inline docker-app/qfieldcloud/settings.py:70]:
```suggestion
    # admin theme
    "jazzmin",
    # django contrib
```

File: docker-app/qfieldcloud/settings.py (hunk 1 above)
Comment (suricactus) [inline docker-app/qfieldcloud/settings.py:598]:
```suggestion
    "custom_css": "css/qfieldcloud.css",
```
Ideally we should only use one CSS file for everything.

**Review by suricactus — APPROVED** [review]
Thanks for contributing this upstream!

Few comments, I know you might not manage to address them yourself:

- [x] Ideally we should not have multiple CSS, if needed we can create `css/admin.css`
- [x] We need to move the favicon to be available in the `core` app

**Review by suricactus — COMMENTED** [review]
Thanks for contributing this upstream!

Few comments, I know you might not manage to address them yourself:

- [ ] Ideally we should not have multiple CSS, if needed we can create `css/admin.css`
- [ ] We need to move the favicon to be available in the `core` app

**Review by suricactus — CHANGES_REQUESTED** [review]
See my comments above before merging.

**Comment by why-not-try-calmer** [comment]
On second thought, shouldn't this PR be made against the private repo, where you do have staticfiles?

**Review by suricactus — APPROVED** [review]
Good move!

**Comment by dddpt** [comment]
Ready for re-review!

**Review by suricactus — APPROVED** [review]
Thank you very much for detecting and fixing untranslated strings in the templates ❤️ .

**Pull request merged**